test/db/cmd/cmd_pd2

```diff
@@ -111,15 +111,15 @@ EXPECT=<<EOF
 |       :   0x00560e67      push  esi
 |      ,==< 0x00560e68      jmp   0x560e7d
 ..
-|      `--> 0x00560e7d      push         ecx
+|      `--> 0x00560e7d      pop   esi
```

Comment:
Wow, does this mean that the push ecx instruction was wrong??

Reply:
Surely you were not expecting a push ecx there right?? :sweat_smile: